server/apps/microservices/src/processors/metadata-extraction.processor.ts

```diff
@@ -257,6 +261,7 @@ export class MetadataExtractionProcessor {
       }
 
       await this.exifRepository.save(newExif);
+      await exiftool.end();
```

Comment:
Adding this line will shutdown the exiftool on other processes. In order to make this work, we have to remove this line. Probably due to the way we are initiating the exif-vendor instance. 

I tested this from uploading from the mobile app, continuously for about 200 assets

Reply:
Which commit was the one you tested with? I just a moments ago pushed more changes, like the way we are initializing the instance for it.

Reply:
Got it. The recent commit works